docs/intro/tutorial02.txt

```diff
@@ -24,42 +24,6 @@ database. When starting your first real project, however, you may want to use a
 more scalable database like PostgreSQL, to avoid database-switching headaches
 down the road.
```

Comment:
I would end this paragraph something like:

> ... down the road (see the database settings reference https://docs.djangoproject.com/en/5.0/ref/settings/#databases for more information).

Reply:
I've included another sentence with links at the end; let me know how this looks @evildmp @timgraham 

https://github.com/django/django/pull/18253/files